.travis.yml

```diff
@@ -20,9 +22,11 @@ install:
   # Install and compile libsvm and liblinear
   - sudo apt-get install -y build-essential
   - git clone https://github.com/cjlin1/libsvm
-  - cd libsvm; make lib; sudo cp libsvm.so.2 /lib; sudo ln -s /lib/libsvm.so.2 /lib/libsvm.so; cd ..
+  - cd libsvm; git checkout v322
+  - make lib; sudo cp libsvm.so.2 /lib; sudo ln -s /lib/libsvm.so.2 /lib/libsvm.so; cd ..
```

Comment:
Now that I think, the CI could actually rely on the compiled Ubuntu binaries introduced by Xsardas1000